js/views/endScreen.js

```diff
@@ -34,6 +34,12 @@ var EndScreen = React.createClass({
     }
   },
 
+  blurOnMouseUp: function(evt) {
+    if (evt.currentTarget) {
+      evt.currentTarget.blur();
```

Comment:
Do we need to check if the blur function exists or is it guaranteed to be there?